lib/prmd/commands/combine.rb

```diff
@@ -119,7 +119,8 @@ def self.combine(paths, options = {})
         end
         meta ||= {}
       end
-      combiner = Prmd::Combiner.new(meta: meta, base: base, schema: schema)
+      type_as_string = options[:type_as_string] || false
+      combiner = Prmd::Combiner.new(meta: meta, base: base, schema: schema, type_as_string: type_as_string)
```

Comment:
maybe the final arg should just be something like `options`, of which `type_as_string` is the first/only one presently, but so we can more easily add/adapt in the future (presumably we can also default to `{}` so that the old method signature would "just-work" also).